package.json

```diff
@@ -11,7 +11,7 @@
     "build": "tsc --build src",
     "ts-jest": "ts-jest",
     "format": "prettier --write src",
-    "prepublishOnly": "npm run build"
+    "prepare": "npm run build"
```

Comment:
Won't this run on every `npm i`? Is building necessary on every installation? When developing I want to `npm i` but don't need to incur a build until I'm ready to distribute/publish

Reply:
Totally up to you. I think it's easier to integrate this package in other packages using prepare.

Reply:
Got it thanks. Yea I think I'll opt to leave it as prepublish for now